Resideo_eshopping/resideo_eshopping/lib/main.dart

```diff
@@ -9,103 +10,10 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Flutter Demo',
```

Comment:
Change the title

Reply:
Changed the title from 'Flutter Demo' to 'Resideo e-Shopping'.